SimG4Core/Geometry/interface/DDG4SolidConverter.h

```diff
@@ -44,8 +46,11 @@ class DDG4SolidConverter
     static const std::vector<double>* par_;
     std::map<DDSolidShape,FNPTR> convDispatch_;
 
+    static G4RotationMatrix* rot;
```

Comment:
@davidlange6 , because it was static and is used by only one static method, if not static it should be likely the member of the class and not its pointer.